### ISSUE TITLE
About modifying the network structure   

### ISSUE THREAD

**Issue by zzzzyq-18** [chenxin-dlut/TransT#143]
Hello author, I tried to add a three-layer perceptron behind the backbone network, and I can find that the training effect is from poor to good, and the loss and iou are normal, but the test effect is very poor (even if it is tested with the training set). May I ask what should be modified after the network structure is modified?
